modules/ROOT/pages/amazon/amazon-s3-connector-design-center.adoc

```diff
@@ -0,0 +1,92 @@
+endif::[]
+:imagesdir: ../../assets/images/
+
+Design Center enables you to create apps visually. To use Design Center, work with your Anypoint Platform administrator to ensure that you have a xref:access-management::environments.adoc#to-create-a-new-environment[Design environment]. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To create an app in Design Center:
+
+* Configure the input source (trigger) for your app.
+* Add the connector as a component to process the input for the app.
+
+For information about Amazon S3 fields, see the xref:amazon/amazon-s3-connector-reference.adoc[Amazon S3 Connector Reference].
+
+== Configure the Input Source Trigger
+
+To configure a trigger:
+
+. In Design Center, click *Create*.
+. Click *Create new application*.
+. Specify a value for *Project name*, and click *Create*.
+. Exit from *Let's get started* by clicking *Go straight to canvas*.
+. Click on the Trigger card.
+. If you are using the connector as an input source trigger, search for the connector and select an operation. Otherwise, search for and select *HTTP* or *Scheduler*.
+. Complete the fields with the values you want to configure for your trigger.
++
+You can use the following Amazon S3 operations as triggers:
++
+* *On Deleted Object* - Initiates access to your app when an Amazon S3 object is deleted.
+* *On New Object* - Initiates access to your app when an Amazon S3 object is created.
++
+Alternatively, you can specify the trigger as an HTTP Listener as input to your application from a browser or an application like Postman or curl, or you can use Scheduler to provide input to your application at a specific time.
+
+== Configure the Target Component
+
+. Click “+” next to the trigger card.
+. In *Select a component*, search for and select the connector name.
+. Select an operation for the connector.
+. Enter the required values in the *General* tab.
+. If you are:
++
+** Using the default Amazon S3 storage, leave the default entries for the *Proxy* and the *Advanced* tabs.
+** Connecting to a different storage than the default AWS S3, specify its URL in the the *Advanced* tab's *S3 Compatible Storage URL* field.
++
+. If needed, enter values for other tabs.
+. Specify access information to the connector resource, as described below.
+. Click *Test* to test the connection.
++
+Validating a connection with Test Connection requires that you have permission in AWS IAM to the action s3:ListAllMyBuckets. If you don’t have this permission, the test fails. However, you can still use the connector and the operations to which you have access.
++
+Access to operations on Amazon S3 is further controlled through policies. It is not always possible to validate your credentials before the exact operation for which you have access completes. This can vary based on the bucket name and other parameters. For example, the test connection can fail if your credentials have a restricted policy.
+
+
```

Comment:
Need a transition here. Users don't see the Connection Type until their 3rd screen: Trigger Card > Set up > Connection Type. Just launching into discussing Connection Types is too hard to follow. Remember too that Design Center users are considered starting users.